### PR TITLE
Css product detail page

### PR DESCRIPTION
**Pull request by priyankaarkade**
- Updated product detail page with css styles.
- In Products page added header and category nav bar.
- used flex in product info details and aligned.
- Let me know if i need to change or refactor css.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,4 +1,100 @@
...
-  height: 100%;
+
+.App {
+  margin: 50px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
```
Comment (kyle-arkade) [inline src/App.css]: Who is living here?

File: src/App.css [hunk 2]
```diff
@@ -1,4 +1,100 @@
...
+li.nav__arkade {
+  text-align: right;
+}
+
+.nav__search, .nav__account, .nav__arkade, .nav__viewcart {  
+  flex: 1;  
+}
+
+@media all and (max-width: 600px) {  
+
```
Comment (kyle-arkade) [inline src/App.css]: Who's house is this?

File: src/containers/ProductDetail/index.js [hunk 3]
```diff
@@ -20,11 +20,14 @@ class ProductDetail extends Component {
             this.setState({
                 product: product
             })
+            console.log(product);
```
Comment (kyle-arkade) [inline src/containers/ProductDetail/index.js]: Nice console log you've got there

File: src/App.css (hunk 1 above)
Comment (priyankaarkade) [reply src/App.css]: removed blank line above the css.

File: src/App.css (hunk 2 above)
Comment (priyankaarkade) [reply src/App.css]: remove blank line below and about the css.

File: src/containers/ProductDetail/index.js (hunk 3 above)
Comment (priyankaarkade) [reply src/containers/ProductDetail/index.js]: haha removed the console.

**Pull request merged**